neigh_linux.go

```diff
@@ -243,7 +243,7 @@ func (h *Handle) NeighListFiltered(linkIndex, family, flags, state int) ([]Neigh
 	return res, nil
 }
 
-func NeighDeserialize(m []byte) (*Neigh, error) {
+func neighDeserialize(m []byte) (*Neigh, error) {
```

Comment:
I agree with this, but unfortunately we release `1.0.0` which had this method exported (this was there for 4 years).
Unexporting it would be a breaking change. We'd prefer avoiding a major release change for this.